build-system/compile/json-schema/README.md

```diff
@@ -0,0 +1,50 @@
+# Using JSON Schema to validate input
+
+## Usage
+
+A file may import a [JSON Schema](https://json-schema.org/) validation function by importing its schema file.
+
+### Example
+
+#### `product.schema.json`
+
+> **Note:** the name of the imported file must end with `.schema.json`
+
+```json
+{
+  "properties": {
+    "title": {"type": "string"}
+  },
+  "required": ["title"]
+}
+```
+
+#### `source.js`
+
+```js
+import validateProduct from './product.schema.json';
```

Comment:
You mean `assert { type: 'json' }`? Would this be correct if we what we're actually importing is a JS function?

Reply:
I see. We would need to update `babel-plugin-transform-json-import` so that it doesn't error with a non-`json` type. 

Alternatively, we could move all this resolution logic into that babel plugin, and get rid of the [`onLoad` issue](https://github.com/ampproject/amphtml/pull/37794#discussion_r832739543) altogether.